src/controllers/krate/metadata.rs

```diff
@@ -220,18 +222,21 @@ pub fn reverse_dependencies(req: &mut dyn Request) -> CargoResult<Response> {
 
     let version_ids: Vec<i32> = rev_deps.iter().map(|dep| dep.version_id).collect();
 
+    //  FIXME   ::  Another one of these silly transformation that is necessary to test
+    //              whether or not this function uses the column email from the user
+    //              table. Here, we simply set it to None.
```

Comment:
Does this comment still apply, since the transformation it's talking about was removed in https://github.com/rust-lang/crates.io/pull/1911/commits/ea51376af883fdc9b12374eda436841a5e15dd09?

Reply:
You are right, I don't think this applies anymore. Do you want me to merge these commits into one?